node-test/src/test/scala/org/bitcoins/node/networking/peer/MerkleBuffersTest.scala

```diff
@@ -41,6 +42,7 @@ class MerkleBuffersTest extends BitcoinSUnitTest {
                 assert(txs == merkleTxs,
                        "Received TXs in callback was not the ones we put in")))
             callbackCount = callbackCount + 1
+            Future.successful(())
```

Comment:
nit: `FutureUtil.unit`

Reply:
`Future.unit` isn't available for scala 2.11.12